docs/site/Appsody-LoopBack.md

```diff
@@ -0,0 +1,990 @@
+   stack (template) listed
+
+   ```
+   REPO            ID                              VERSION         TEMPLATES               DESCRIPTION
+
+   ...
+   *incubator      nodejs-loopback                 0.1.7           *scaffold               LoopBack 4 API Framework for Node.js
+   ...
+   ```
+
+5. Scaffold a LoopBack 4 application in the current directory with:
+
+   ```sh
+   appsody init nodejs-loopback
+   ```
+
+   You should see output like this:
+
+   ```
+   Checking stack requirements...
+   Skipping Buildah - No requirements set.
+   Skipping Docker - No requirements set.
+   Skipping Appsody - No requirements set.
+   Running appsody init...
+   Downloading nodejs-loopback template project from https://github.com/appsody/stacks/releases/download/nodejs-loopback-v0.1.7/incubator.nodejs-loopback.v0.1.7.templates.scaffold.ta
+   r.gz
+   Download complete. Extracting files from /appsodyLB4Todo/nodejs-loopback.tar.gz
+   Setting up the development environment
+   Your Appsody project name has been set to appsodylb4todo
+   Pulling docker image appsody/nodejs-loopback:0.1
+   Running command: docker pull appsody/nodejs-loopback:0.1
+   0.1: Pulling from appsody/nodejs-loopback
+   ...
+   Status: Downloaded newer image for appsody/nodejs-loopback:0.1
+   docker.io/appsody/nodejs-loopback:0.1
+   [Warning] The stack image does not contain APPSODY_PROJECT_DIR. Using /project
+   Running command: docker run --rm --entrypoint /bin/bash appsody/nodejs-loopback:0.1 -c find /project -type f -name .appsody-init.sh
+   Successfully initialized Appsody project with the nodejs-loopback stack and the default template.
+   ```
+
+   The most important lines to make note of are:
+
+   ```
+   Your Appsody project name has been set to appsodylb4todo
+
+   and
+
+   Successfully initialized Appsody project with the nodejs-loopback stack and the default template.
+   ```
+
+6. Install all the dependencies of this application with:
+
+   ```sh
+   npm install
+   ```
+
+7. Now that all the dependencies have been installed, it is a perfect time to
+   open your favourite IDE editor on this project.
+
+   In my case, I using VS Code.
+
+   ```sh
+   code .
+   ```
+
+8. The basic structure of the application looks like this:
+
+   ![lb4appsody_template_files_1.png](./imgs/lb4appsody_template_files_1.png)
+
+   One important file is the Appsody configuration file for your project named
+   **.appsody-config.yaml** .
+
+   It defines the **name** of your project, and the **stack** (template) on
+   which it is based.
+
+   ```yaml
+   project-name: appsodylb4todo
+   stack: appsody/nodejs-loopback:0.1
+   ```
+
+9. The source code of the LoopBack 4 application is located in the **src**
+   directory.
+
+   ![lb4appsody_template_files_2.png](./imgs/lb4appsody_template_files_2.png)
+
+10. This basic application only defines one API endpoint `'/ping'` in the file
+    **src/controllers/ping.controller.ts**
```

Comment:
We can mention the initial application is very much similar as the one generated by `lb4 app` command.